lib/player.js

```diff
@@ -6354,46 +6429,43 @@ shaka.Player = class extends shaka.util.FakeEventTarget {
       return;
     }
 
-    let liveSyncMaxLatency;
-    let liveSyncPlaybackRate;
-    let liveSyncMinLatency;
-    let liveSyncMinPlaybackRate;
-    const liveSyncTargetLatencyTolerance =
-      this.config_.streaming.liveSyncTargetLatencyTolerance;
-
-    if (this.config_.streaming.liveSync) {
-      const liveSyncTargetLatency =
-        this.config_.streaming.liveSyncTargetLatency;
-      liveSyncMaxLatency =
-        liveSyncTargetLatency + liveSyncTargetLatencyTolerance;
-      liveSyncMinLatency = Math.max(0,
-          liveSyncTargetLatency - liveSyncTargetLatencyTolerance);
-      liveSyncPlaybackRate = this.config_.streaming.liveSyncPlaybackRate;
-      liveSyncMinPlaybackRate = this.config_.streaming.liveSyncMinPlaybackRate;
+    let maxLatency;
+    let maxPlaybackRate;
+    let minLatency;
+    let minPlaybackRate;
+    const targetLatencyTolerance =
```

Comment:
removed `liveSync` from these to have shorter names. `liveSyncPlaybackRate` became `maxPlaybackRate` to not override the current playbackRate var.

The shorter names allowed me to combine some lines below.

Reply:
Thanks!

Reply:
I will rename `streaming.liveSync.playbackRate` to `maxPlaybackRate`